MVCapplication/src/main/java/com/softserve/if072/mvcapp/dto/StoresInProduct.java

```diff
@@ -13,4 +13,19 @@
     public void setStoresId(List<Integer> storesId) {
         this.storesId = storesId;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00122)